src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -276,7 +276,7 @@ export default {
 		:type="isButtonType ? 'button' : null"
 		v-on="isButtonType ? listeners : null">
 		<input v-if="!isButtonType"
-			:id="id"
+			:aria-labelledby="!isButtonType && !ariaLabel ? id : null"
```

Comment:
Isn't it a breaking change?

This component has a prop `id` which still has description `Unique id attribute of the input`.

But not an instance with `id` set won't render an `input` with this `id` but a `span`.

Example of breaking change:
```html
<!-- label doesn't work anymore -->
<label for="my-id">Custom label outside</label>

<NcCheckboxRadioSwitch id="my-id" />

<script>
// Doesn't work
document.getElementById('my-id').focus()
</script>
```

Maybe examples are edge cases, but they are supposed to work according to the current description `id attribute of the input`

And the same with `NcCheckboxContent` where the meaning of `id` prop has changed.

At least I'd change the prop descriptions.

---

As an alternative, I may propose to not use `id` from prop here, but generate a new local `id` and pass it to the `NcCheckboxContent` as a new prop.

Reply:
Ah, right, I used the wrong id maybe, I'll have a second look this morning! Thanks @ShGKme  :)

Reply:
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/14975046/be91d485-b8ca-4d45-b602-451484d9c2da)

Here we go